modules/nw-cluster-network-range-edit.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-range.adoc
+
+:_content-type: PROCEDURE
+[id="nw-cluster-network-range-edit_{context}"]
+= Expanding the cluster network IP address range
+
+You can expand the IP address range for the cluster network.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Ensure that the cluster uses the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. To obtain the cluster network range and host prefix for your cluster, enter the following command:
++
+[source,terminal]
+----
+$ oc get network.operator.openshift.io \
+  -o jsonpath="{.items[0].spec.clusterNetwork}"
+----
++
+.Example output
+[source,text]
+----
+[{"cidr":"10.217.0.0/22","hostPrefix":23}]
+----
+
+. To expand the cluster network IP address range, enter the following command. Use the CIDR IP address range and host prefix returned from the output of the previous command.
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster --type='merge' --patch \
+  '{
+    "spec":{
+      "clusterNetwork": [ {"cidr":"<cidr>","hostPrefix":<prefix>} ],
+      "networkType": "OVNKubernetes"
+    }
+  }'
+----
++
+--
+where:
+
+`<cidr>`: Specifies the cluster network IP address range. You can change the CIDR mask. Do not change the IP address range. For example, `10.217.0.0/14`.
```

Comment:
You can only change the CIDR mask to a lower number.  Do not change the network IP address in the CIDR.